lib/webtask.js

```diff
@@ -797,3 +879,27 @@ function parse_ten(ten) {
 function normalize_host(host) {
     return (host ? host.toLowerCase().replace(/\:/g, '-') : 'global');
 }
+
+function wants_html(req) {
```

Comment:
The logic here should be to determine if the request _prefers_ html. An `Accepts` header can list a number of different mime-types (or wild-cards) with associated priorities.

That being said, in theory, a webtask can return anything it wants. This would mean we can't be sure what relative preference to check against `html`. Perhaps you are right in just asking if it will accept `html` here. What do you think?